EventsExpress/ClientApp/src/actions/event/event-add-action.js

```diff
@@ -34,7 +34,7 @@ export function edit_event(data) {
             throw new SubmissionError(await buildValidationState(response));
         }
         dispatch(setEventSuccess(true));
-        dispatch(getEvent(data));
+        dispatch(get_event(data.id));
```

Comment:
Yes, indeed. Because this method already worked well. Probably, someone might have mistakenly changed its name and the parameter passed to this method. I can assume that this is due to the presence of two almost identical methods which have similar names. Nevertheless, this function requires passing only the identifier to this method, not the entire data. That's why I used exactly the "get_event" method.

Reply:
@alexanchez, could you please find from the history which issue it fixed and post it here? Also try your reproducing that issue with your fix please.